Open Judge System/Services/OJS.Services/Business/Participants/ParticipantsBusinessService.cs

```diff
@@ -146,6 +152,26 @@ public void UpdateTotalScoreSnapshotOfParticipants()
             this.participantsData.UpdateTotalScoreSnapshot();
         }
 
+        public void RemoveParticipantMultipleScores()
+        {
+            var participants = 
+                this.scoresDataService.GetAll()
+                .GroupBy(ps => new { ps.IsOfficial, ps.ProblemId, ps.ParticipantId })
+                .Where(ps=>ps.Count() > 1)
+                .ToList();
+            
+            using (var scope = TransactionsHelper.CreateTransactionScope())
+            {
+                foreach (var participant in participants)
+                {
+                    var participantsToRemove = participant.OrderByDescending(ps => ps.Points).Skip(1).ToList();
+                    this.scoresDataService.Delete(participantsToRemove);
```

Comment:
This will make separate db request for each participant. Can we make it more efficient, by deleting all at once or in batches (but provided they are not expected to be many, I say at once)